frontend/src/components/comments/CommentList.jsx

```diff
@@ -1,14 +1,30 @@
 // NPM Packages
-import React from "react";
+import React from 'react';
 
 // Project files
-import CommentCard from "./CommentCard";
+import CommentCard from './CommentCard';
 
-export default function CommentList({ postId, comments, onDelete, currentUser }) {
+export default function CommentList({
+  postId,
+  comments,
+  onDelete,
+  currentUser,
+}) {
   // Components
   const CommentCardArray = comments.map((comment) => (
-    <CommentCard key={comment.id} postId={postId} comment={comment} currentUser={currentUser} onDeleteClick={onDelete} />
+    <CommentCard
+      key={comment.id}
+      postId={postId}
+      comment={comment}
+      currentUser={currentUser}
+      onDeleteClick={onDelete}
+    />
   ));
 
-  return <div className="CommentList">{CommentCardArray}</div>;
+  return (
+    <details className="CommentList" open>
```

Comment:
Should we have the list hidden by default and therefore remove `open`?

Reply:
great idea !